packages/react-test-renderer/shallow.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  throw Error('shallow renderer is not available in production mode.');
+} else {
```

Comment:
Should we also throw for the main test renderer entry point then?

Reply:
@gaearon @bvaughn I'm having a problem with these lines: I'm using [enzyme](https://github.com/airbnb/enzyme) and it uses shallow rendering, _for tests_, while `NODE_ENV` is set to `production`. So my tests are failing because of this.